### ISSUE TITLE
[Web] JSNoSuchMethodError: Cannot read properties of undefined (reading 'appTheme')

### ISSUE THREAD

**Issue by wartelski** [flutter/flutter#145410]
### Steps to reproduce

1. start the app
2. turn off wifi
3. open the dialog/popup with GoogleMap widget

### Expected results

not crash just because a network request fails

### Actual results

It's throwing an exception 
The following JSNoSuchMethodError was thrown building GoogleMap(dirty, dependencies: [Directionality], state: _GoogleMapState#1f2e8):
TypeError: Cannot read properties of undefined (reading 'maps')

### Code sample

<details open><summary>Code sample</summary>

```dart
class LocationMap extends StatefulWidget {
  final LatLng? selectedLocation;

  const LocationMap({
    Key? key,
    this.selectedLocation,
  }) : super(key: key);

  @override
  State<LocationMap> createState() => _LocationMapState();
}

class _LocationMapState extends State<LocationMap> {
  GoogleMapController? mapController;

  @override
  Widget build(BuildContext context) {
    LatLng initialCameraPosition = widget.selectedLocation;


     if (mapController != null && widget.selectedLocation != null) {
       mapController!.animateCamera(CameraUpdate.newCameraPosition(
         CameraPosition(
           target: widget.selectedLocation!,
           zoom: 11.0,
         ),
       ));
     }

    return Card(
       child: GoogleMap(
         onMapCreated: (controller) {
           mapController = controller;
         },
         initialCameraPosition: CameraPosition(
           target: initialCameraPosition,
           zoom: 11.0,
         ),
         markers: widget.selectedLocation != null
             ? {
           Marker(
             markerId: MarkerId('selectedLocation'),
             position: widget.selectedLocation!,
           ),
         }
             : {},
       ),
     );
  }
}

```

</details>

### Screenshots or Video

<details open>
<summary>Screenshots / Video demonstration</summary>

[Upload media here]

</details>
<img width="529" alt="Screenshot 2024-03-12 at 6 22 44 PM" src="https://github.com/flutter/flutter/assets/131498294/2309ec7c-9e64-4ed9-8c51-a66669d3679f">

### Logs

<details open><summary>Logs</summary>

```console
======== Exception caught by widgets library =======================================================
The following JSNoSuchMethodError was thrown building GoogleMap(dirty, dependencies: [Directionality], state: _GoogleMapState#1f2e8):
TypeError: Cannot read properties of undefined (reading 'maps')

The relevant error-causing widget was: 
  GoogleMap GoogleMap:file:///Users/anastasia_wartell/rwsecure-flutter/lib/screens/components/location_input/location_map.dart:41:15
When the exception was thrown, this was the stack: 
packages/google_maps_flutter_web/src/convert.dart 100:30                          _gmapTypeIDForPluginType
packages/google_maps_flutter_web/src/convert.dart 43:25                           _configurationAndStyleToGmapsOptions
packages/google_maps_flutter_web/src/google_maps_controller.dart 173:32           init
packages/google_maps_flutter_web/src/google_maps_flutter_web.dart 311:8           <fn>
packages/google_maps_flutter_web/src/google_maps_flutter_web.dart 311:8           buildViewWithConfiguration
packages/google_maps_flutter/src/google_map.dart 321:47                           build
packages/flutter/src/widgets/framework.dart 5409:27                               build
packages/flutter/src/widgets/framework.dart 5297:15                               performRebuild
packages/flutter/src/widgets/framework.dart 5462:11                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5453:11                               [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5462:11                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5453:11                               [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5462:11                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5453:11                               [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5462:11                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5453:11                               [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5462:11                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5453:11                               [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 6569:36                               inflateWidget
packages/flutter/src/widgets/framework.dart 6581:32                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 6569:36                               inflateWidget
packages/flutter/src/widgets/framework.dart 6581:32                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5462:11                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5453:11                               [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5462:11                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5453:11                               [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5462:11                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5453:11                               [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5462:11                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5453:11                               [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 6569:36                               inflateWidget
packages/flutter/src/widgets/framework.dart 6581:32                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5462:11                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5453:11                               [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5462:11                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5453:11                               [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5462:11                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5453:11                               [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5462:11                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5453:11                               [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 6434:14                               mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/framework.dart 5322:16                               performRebuild
packages/flutter/src/widgets/framework.dart 5016:7                                rebuild
packages/flutter/src/widgets/framework.dart 5279:5                                [_firstBuild]
packages/flutter/src/widgets/framework.dart 5273:5                                mount
packages/flutter/src/widgets/framework.dart 4182:15                               inflateWidget
packages/flutter/src/widgets/framework.dart 3707:18                               updateChild
packages/flutter/src/widgets/layout_builder.dart 135:18                           layoutCallback
packages/flutter/src/widgets/framework.dart 2719:19                               buildScope
packages/flutter/src/widgets/layout_builder.dart 153:5                            [_layout]
packages/flutter/src/rendering/object.dart 2604:59                                <fn>
packages/flutter/src/rendering/object.dart 1059:15                                [_enableMutationsToDirtySubtrees]
packages/flutter/src/rendering/object.dart 2604:7                                 invokeLayoutCallback
packages/flutter/src/widgets/layout_builder.dart 228:7                            rebuildIfNecessary
packages/flutter/src/widgets/layout_builder.dart 313:5                            performLayout
packages/flutter/src/rendering/object.dart 2493:7                                 layout
packages/flutter/src/rendering/box.dart 2382:11                                   layout
packages/flutter/src/rendering/shifted_box.dart 1291:7                            performLayout
packages/flutter/src/rendering/object.dart 2493:7                                 layout
packages/flutter/src/rendering/box.dart 2382:11                                   layout
packages/flutter/src/rendering/shifted_box.dart 238:5                             performLayout
packages/flutter/src/rendering/object.dart 2493:7                                 layout
packages/flutter/src/rendering/box.dart 2382:11                                   layout
packages/flutter/src/rendering/proxy_box.dart 104:21                              <fn>
packages/flutter/src/rendering/object.dart 2493:7                                 layout
packages/flutter/src/rendering/box.dart 2382:11                                   layout
packages/flutter/src/rendering/proxy_box.dart 104:21                              <fn>
packages/flutter/src/rendering/object.dart 2493:7                                 layout
packages/flutter/src/rendering/box.dart 2382:11                                   layout
packages/flutter/src/rendering/proxy_box.dart 104:21                              <fn>
packages/flutter/src/rendering/object.dart 2493:7                                 layout
packages/flutter/src/rendering/box.dart 2382:11                                   layout
packages/flutter/src/rendering/proxy_box.dart 104:21                              <fn>
packages/flutter/src/rendering/object.dart 2493:7                                 layout
packages/flutter/src/rendering/box.dart 2382:11                                   layout
packages/flutter/src/rendering/proxy_box.dart 104:21                              <fn>
packages/flutter/src/rendering/object.dart 2493:7                                 layout
packages/flutter/src/rendering/box.dart 2382:11                                   layout
packages/flutter/src/rendering/proxy_box.dart 104:21                              <fn>
packages/flutter/src/rendering/object.dart 2493:7                                 layout
packages/flutter/src/rendering/box.dart 2382:11                                   layout
packages/flutter/src/rendering/proxy_box.dart 104:21                              <fn>
packages/flutter/src/rendering/proxy_box.dart 3732:13                             performLayout
packages/flutter/src/rendering/object.dart 2493:7                                 layout
packages/flutter/src/rendering/box.dart 2382:11                                   layout
packages/flutter/src/rendering/proxy_box.dart 104:21                              <fn>
packages/flutter/src/rendering/object.dart 2493:7                                 layout
packages/flutter/src/rendering/box.dart 2382:11                                   layout
packages/flutter/src/widgets/overlay.dart 838:14                                  performLayout
packages/flutter/src/rendering/object.dart 2332:7                                 [_layoutWithoutResize]
packages/flutter/src/rendering/object.dart 1013:17                                flushLayout
packages/flutter/src/rendering/binding.dart 494:19                                drawFrame
packages/flutter/src/widgets/binding.dart 918:13                                  drawFrame
packages/flutter/src/rendering/binding.dart 360:5                                 [_handlePersistentFrameCallback]
packages/flutter/src/scheduler/binding.dart 1297:15                               [_invokeFrameCallback]
packages/flutter/src/scheduler/binding.dart 1227:9                                handleDrawFrame
packages/flutter/src/scheduler/binding.dart 1085:5                                [_handleDrawFrame]
dart-sdk/lib/async/zone.dart 1399:13                                              _rootRun
dart-sdk/lib/async/zone.dart 1301:19                                              run
dart-sdk/lib/async/zone.dart 1209:7                                               runGuarded
lib/_engine/engine/platform_dispatcher.dart 1306:9                                invoke
lib/_engine/engine/platform_dispatcher.dart 278:5                                 invokeOnDrawFrame
lib/_engine/engine/initialization.dart 185:45                                     <fn>
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 574:37  _checkAndCall
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 579:39  dcall
====================================================================================================
```

</details>

### Flutter Doctor output

<details open><summary>Doctor output</summary>

```console
[✓] Flutter (Channel stable, 3.13.7, on macOS 14.3 23D56 darwin-arm64, locale en-US)
    • Flutter version 3.13.7 on channel stable at /.../.../flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 2f708eb839 (5 months ago), 2023-10-09 09:58:08 -0500
    • Engine revision a794cf2681
    • Dart version 3.1.3
    • DevTools version 2.25.0

[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /.../.../Library/Android/sdk
    • Platform android-34, build-tools 34.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b802.4-9586694)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 15.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 15E204a
    • CocoaPods version 1.13.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2022.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b802.4-9586694)

[✓] IntelliJ IDEA Ultimate Edition (version 2023.2.2)
    • IntelliJ at /Applications/IntelliJ IDEA.app
    • Flutter plugin version 76.3.4
    • Dart plugin version 232.9559.10

[✓] VS Code (version 1.84.2)
    • VS Code at /.../.../.../Visual Studio Code.app/Contents
    • Flutter extension can be installed from:
      🔨 https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-arm64   • macOS 14.3 23D56 darwin-arm64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 122.0.6261.129

[✓] Network resources
    • All expected network resources are available.

• No issues found!

```

</details>

**Comment by huycozy** [comment]
Hi @wartelski 
I check the issue using [package's example](https://github.com/flutter/packages/blob/main/packages/google_maps_flutter/google_maps_flutter/example/lib/main.dart) but the error is different, with `net::ERR_CONNECTION_CLOSED` instead.

What is your current maps package version? (the latest version is `google_maps_flutter: ^2.6.0`)

Could you also try this on the latest Flutter stable `3.19.3`?

**Comment by wartelski** [comment]
Hi @huycozy 
Thank you for checking the issue.
It's hard to catch the error sometimes, but after updating to the latest stable Flutter version, it seems like it's fine now. If I will get the error again, I will let you know. Thanks.

**Comment by huycozy** [comment]
Glad to know it. Since the issue doesn't occur currently, I will close the issue. But once you see it appear again, please write in the comments and I will reopen this.

**Comment by polina-c** [comment]
Getting this error regularly on flutter master for [this web app](https://github.com/open-field-language/open-field-language), when running `flutter run -d chrome`.

```
The following JSNoSuchMethodError was thrown building MainApp(dirty):
TypeError: Cannot read properties of undefined (reading 'appTheme')

The relevant error-causing widget was:
  MainApp MainApp:file:///Users/polinach/_/open-field-language/website/lib/main.dart:20:16

When the exception was thrown, this was the stack:
packages/ofl_web/main.dart 30:14                                         build
packages/flutter/src/widgets/framework.dart 5687:22                      build
packages/flutter/src/widgets/framework.dart 5617:15                      performRebuild
packages/flutter/src/widgets/framework.dart 5333:7                       rebuild
packages/flutter/src/widgets/framework.dart 5599:5                       [_firstBuild]
packages/flutter/src/widgets/framework.dart 5593:5                       mount
packages/flutter/src/widgets/framework.dart 4468:15                      inflateWidget
packages/flutter/src/widgets/framework.dart 3963:18                      updateChild
packages/flutter/src/widgets/framework.dart 5642:16                      performRebuild
packages/flutter/src/widgets/framework.dart 5333:7                       rebuild
packages/flutter/src/widgets/framework.dart 5599:5                       [_firstBuild]
packages/flutter/src/widgets/framework.dart 5593:5                       mount
packages/flutter/src/widgets/framework.dart 4468:15                      inflateWidget
packages/flutter/src/widgets/framework.dart 3963:18                      updateChild
packages/flutter/src/widgets/framework.dart 5642:16                      performRebuild
packages/flutter/src/widgets/framework.dart 5780:11                      performRebuild
packages/flutter/src/widgets/framework.dart 5333:7                       rebuild
packages/flutter/src/widgets/framework.dart 5599:5                       [_firstBuild]
packages/flutter/src/widgets/framework.dart 5771:11                      [_firstBuild]
packages/flutter/src/widgets/framework.dart 5593:5                       mount
packages/flutter/src/widgets/framework.dart 4468:15                      inflateWidget
packages/flutter/src/widgets/framework.dart 3963:18                      updateChild
packages/flutter/src/widgets/framework.dart 5642:16                      performRebuild
packages/flutter/src/widgets/framework.dart 5333:7                       rebuild
packages/flutter/src/widgets/framework.dart 5599:5                       [_firstBuild]
packages/flutter/src/widgets/framework.dart 5593:5                       mount
packages/flutter/src/widgets/framework.dart 4468:15                      inflateWidget
packages/flutter/src/widgets/framework.dart 3963:18                      updateChild
packages/flutter/src/widgets/framework.dart 5642:16                      performRebuild
packages/flutter/src/widgets/framework.dart 5780:11                      performRebuild
packages/flutter/src/widgets/framework.dart 5333:7                       rebuild
packages/flutter/src/widgets/framework.dart 5599:5                       [_firstBuild]
packages/flutter/src/widgets/framework.dart 5771:11                      [_firstBuild]
packages/flutter/src/widgets/framework.dart 5593:5                       mount
packages/flutter/src/widgets/framework.dart 4468:15                      inflateWidget
packages/flutter/src/widgets/framework.dart 3963:18                      updateChild
packages/flutter/src/widgets/framework.dart 5642:16                      performRebuild
packages/flutter/src/widgets/framework.dart 5780:11                      performRebuild
packages/flutter/src/widgets/framework.dart 5333:7                       rebuild
packages/flutter/src/widgets/framework.dart 5599:5                       [_firstBuild]
packages/flutter/src/widgets/framework.dart 5771:11                      [_firstBuild]
packages/flutter/src/widgets/framework.dart 5593:5                       mount
packages/flutter/src/widgets/framework.dart 4468:15                      inflateWidget
packages/flutter/src/widgets/framework.dart 3963:18                      updateChild
packages/flutter/src/widgets/framework.dart 5642:16                      performRebuild
packages/flutter/src/widgets/framework.dart 5333:7                       rebuild
packages/flutter/src/widgets/framework.dart 5599:5                       [_firstBuild]
packages/flutter/src/widgets/framework.dart 5593:5                       mount
packages/flutter/src/widgets/framework.dart 4468:15                      inflateWidget
packages/flutter/src/widgets/framework.dart 3963:18                      updateChild
packages/flutter/src/widgets/framework.dart 5642:16                      performRebuild
packages/flutter/src/widgets/framework.dart 5780:11                      performRebuild
packages/flutter/src/widgets/framework.dart 5333:7                       rebuild
packages/flutter/src/widgets/framework.dart 5599:5                       [_firstBuild]
packages/flutter/src/widgets/framework.dart 5771:11                      [_firstBuild]
packages/flutter/src/widgets/framework.dart 5593:5                       mount
packages/flutter/src/widgets/framework.dart 4468:15                      inflateWidget
packages/flutter/src/widgets/framework.dart 3963:18                      updateChild
packages/flutter/src/widgets/framework.dart 5642:16                      performRebuild
packages/flutter/src/widgets/framework.dart 5333:7                       rebuild
packages/flutter/src/widgets/framework.dart 5599:5                       [_firstBuild]
packages/flutter/src/widgets/framework.dart 5593:5                       mount
packages/flutter/src/widgets/framework.dart 4468:15                      inflateWidget
packages/flutter/src/widgets/framework.dart 3963:18                      updateChild
packages/flutter/src/widgets/framework.dart 5642:16                      performRebuild
packages/flutter/src/widgets/framework.dart 5333:7                       rebuild
packages/flutter/src/widgets/framework.dart 5599:5                       [_firstBuild]
packages/flutter/src/widgets/framework.dart 5593:5                       mount
packages/flutter/src/widgets/framework.dart 4468:15                      inflateWidget
packages/flutter/src/widgets/framework.dart 3963:18                      updateChild
packages/flutter/src/widgets/view.dart 441:16                            [_updateChild]
packages/flutter/src/widgets/view.dart 464:5                             mount
packages/flutter/src/widgets/framework.dart 4468:15                      inflateWidget
packages/flutter/src/widgets/framework.dart 3963:18                      updateChild
packages/flutter/src/widgets/framework.dart 5642:16                      performRebuild
packages/flutter/src/widgets/framework.dart 5333:7                       rebuild
packages/flutter/src/widgets/framework.dart 5599:5                       [_firstBuild]
packages/flutter/src/widgets/framework.dart 5593:5                       mount
packages/flutter/src/widgets/framework.dart 4468:15                      inflateWidget
packages/flutter/src/widgets/framework.dart 3963:18                      updateChild
packages/flutter/src/widgets/framework.dart 5642:16                      performRebuild
packages/flutter/src/widgets/framework.dart 5780:11                      performRebuild
packages/flutter/src/widgets/framework.dart 5333:7                       rebuild
packages/flutter/src/widgets/framework.dart 5599:5                       [_firstBuild]
packages/flutter/src/widgets/framework.dart 5771:11                      [_firstBuild]
packages/flutter/src/widgets/framework.dart 5593:5                       mount
packages/flutter/src/widgets/framework.dart 4468:15                      inflateWidget
packages/flutter/src/widgets/framework.dart 3963:18                      updateChild
packages/flutter/src/widgets/binding.dart 1605:16                        [_rebuild]
packages/flutter/src/widgets/binding.dart 1574:5                         mount
packages/flutter/src/widgets/binding.dart 1527:16                        <fn>
packages/flutter/src/widgets/framework.dart 3038:11                      buildScope
packages/flutter/src/widgets/binding.dart 1526:12                        attach
packages/flutter/src/widgets/binding.dart 1265:26                        attachToBuildOwner
packages/flutter/src/widgets/binding.dart 1247:5                         attachRootWidget
packages/flutter/src/widgets/binding.dart 1233:7                         <fn>
dart-sdk/lib/_internal/js_dev_runtime/private/isolate_helper.dart 48:11  internalCallback
```

```
Flutter 3.23.0-14.0.pre.75 • channel master • git@github.com:flutter/flutter.git
Framework • revision 38af6be9fb (5 days ago) • 2024-06-29 13:06:24 -0700
Engine • revision ad1343cfb9
Tools • Dart 3.5.0 (build 3.5.0-315.0.dev) • DevTools 2.36.0
```

**Comment by polina-c** [comment]
Refresh of chrome helps sometimes

**Comment by huycozy** [comment]
I re-checked the original issue (using maps plugin and disabling internet) with the latest package version `google_maps_flutter: ^2.7.0` but only see network error on browser console:

```console
POST https://maps.googleapis.com/$rpc/google.internal.maps.mapsjs.v1.MapsJsInternalService/GetViewportInfo net::ERR_INTERNET_DISCONNECTED
```

However, I can reproduce the issue with sample code @polina-c provided on Flutter master channel without needing to disable the internet.

I don't see the issue on Flutter stable 3.22.2 and on a simple Flutter project. The [2nd sample code](https://github.com/open-field-language/open-field-language) is a complex project but the stack trace may be enough to investigate further, labeling the issue for others input. 

<details>
<summary>flutter doctor -v (stable and master)</summary>

```bash
[✓] Flutter (Channel stable, 3.22.2, on macOS 14.1 23B74 darwin-x64, locale en-VN)
    • Flutter version 3.22.2 on channel stable at /Users/huynq/Documents/GitHub/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 761747bfc5 (3 weeks ago), 2024-06-05 22:15:13 +0200
    • Engine revision edd8546116
    • Dart version 3.4.3
    • DevTools version 2.34.3

[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /Users/huynq/Library/Android/sdk
    • Platform android-34, build-tools 34.0.0
    • ANDROID_HOME = /Users/huynq/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.10+0-17.0.10b1087.21-11572160)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 15.4)
    • Xcode at /Applications/Xcode15.4.app/Contents/Developer
    • Build 15F31d
    • CocoaPods version 1.15.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2023.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • android-studio-dir = /Applications/Android Studio.app/
    • Java version OpenJDK Runtime Environment (build 17.0.10+0-17.0.10b1087.21-11572160)

[✓] VS Code (version 1.90.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.90.0

[✓] Connected device (3 available)
    • iPhone (mobile) • d9a94afe2b649fef56ba0bfeb052f0f2a7dae95e • ios            • iOS 15.8 19H370
    • macOS (desktop) • macos                                    • darwin-x64     • macOS 14.1 23B74 darwin-x64
    • Chrome (web)    • chrome                                   • web-javascript • Google Chrome 126.0.6478.127

[✓] Network resources
    • All expected network resources are available.

• No issues found!
```

```bash
[!] Flutter (Channel master, 3.23.0-14.0.pre.145, on macOS 14.1 23B74 darwin-x64, locale en-VN)
    • Flutter version 3.23.0-14.0.pre.145 on channel master at /Users/huynq/Documents/GitHub/flutter_master
    ! Warning: `flutter` on your path resolves to /Users/huynq/Documents/GitHub/flutter/bin/flutter, which is not inside your current Flutter SDK checkout at /Users/huynq/Documents/GitHub/flutter_master. Consider adding /Users/huynq/Documents/GitHub/flutter_master/bin to the front of your path.
    ! Warning: `dart` on your path resolves to /Users/huynq/Documents/GitHub/flutter/bin/dart, which is not inside your current Flutter SDK checkout at /Users/huynq/Documents/GitHub/flutter_master. Consider adding /Users/huynq/Documents/GitHub/flutter_master/bin to the front of your path.
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ac74e26a70 (2 hours ago), 2024-07-04 21:46:34 -0400
    • Engine revision 1f0f950ea0
    • Dart version 3.5.0 (build 3.5.0-323.0.dev)
    • DevTools version 2.37.0
    • If those were intentional, you can disregard the above warnings; however it is recommended to use "git" directly to perform update checks and upgrades.

[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /Users/huynq/Library/Android/sdk
    • Platform android-34, build-tools 34.0.0
    • ANDROID_HOME = /Users/huynq/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.10+0-17.0.10b1087.21-11572160)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 15.4)
    • Xcode at /Applications/Xcode15.4.app/Contents/Developer
    • Build 15F31d
    • CocoaPods version 1.15.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2023.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • android-studio-dir = /Applications/Android Studio.app/
    • Java version OpenJDK Runtime Environment (build 17.0.10+0-17.0.10b1087.21-11572160)

[✓] VS Code (version 1.90.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.92.0

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-x64     • macOS 14.1 23B74 darwin-x64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 126.0.6478.127

[✓] Network resources
    • All expected network resources are available.

! Doctor found issues in 1 category.
```
</details>

**Comment by huycozy** [comment]
When checking https://github.com/flutter/flutter/issues/152844 on a simple app (counter app) on Flutter beta channel `3.24.0-0.2.pre`, I see a similar error appearing randomly (sometime the error appears, sometime not):

```console
TypeError: Cannot read properties of undefined (reading 'Symbol(_privateNames)')
```

```console
TypeError: Cannot read properties of undefined (reading 'Colors')
```

**Comment by ditman** [comment]
@huycozy do you see the issue in `master`? I know there was a fix added to how things are loaded that might not have landed in the beta preview:

* https://github.com/flutter/engine/pull/53718

**Comment by huycozy** [comment]
@ditman I don't see this issue on master `3.24.0-1.0.pre.460` anymore.

**Comment by mdebbar** [comment]
This is fixed in master and will be fixed in stable once these cherrypicks land: https://github.com/flutter/flutter/issues/152953

**Comment by Hami0095** [comment]
This branch should be reopened, because, I am getting the Exception: 
═╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞═══════════════════════════════════════════════════════════
The following JSNoSuchMethodError was thrown building MyApp(dirty):
TypeError: Cannot read properties of undefined (reading 'Colors') for the demo Counter Application. 

Flutter's Version is :

Flutter is already up to date on channel stable
Flutter 3.24.0 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 80c2e84975 (3 weeks ago) • 2024-07-30 23:06:49 +0700
Engine • revision b8800d88be
Tools • Dart 3.5.0 • DevTools 2.37.2

**Comment by wartelski** [comment]
Hello,
after updating flutter to 3.24 I started to get a bunch of errors after starting the app (Flutter Web). Before the update, everything was working perfectly, and I didn't make any significant changes in my code, so I think it's connected with the update. It seems like the issue is mainly with loading packages, but I could be mistaken.
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.24.0, on macOS 14.5 23F79 darwin-arm64, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 15.4)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2022.2)
[✓] IntelliJ IDEA Ultimate Edition (version 2024.1.4)
[✓] VS Code (version 1.84.2)
[✓] Connected device (4 available)
[✓] Network resources

• No issues found!
```
Errors:
```
Error: google_fonts was unable to load font SourceSans3-Regular because the following exception occurred:
TypeError: Cannot read properties of undefined (reading 'SynchronousFuture$')
If troubleshooting doesn't solve the problem, please file an issue at https://github.com/material-foundation/flutter-packages/issues/new/choose.

```
Another one:
```
======== Exception caught by widgets library =======================================================
The following JSNoSuchMethodError was thrown building _FocusInheritedScope:
TypeError: Cannot read properties of undefined (reading 'SynchronousFuture$')

The relevant error-causing widget was: 
  MaterialApp MaterialApp:file:///.../.../.../lib/main.dart:79:28
When the exception was thrown, this was the stack: 
packages/flutter_gen/gen_l10n/app_localizations.dart 756:12              load
packages/flutter/src/widgets/localizations.dart 57:48                    _loadAll
packages/flutter/src/widgets/localizations.dart 590:8                    load
packages/flutter/src/widgets/localizations.dart 554:5                    initState
packages/flutter/src/widgets/framework.dart 5748:55                      [_firstBuild]
packages/flutter/src/widgets/framework.dart 5593:5                       mount
packages/flutter/src/widgets/framework.dart 4468:15                      inflateWidget
packages/flutter/src/widgets/framework.dart 3963:18                      updateChild
```
<img width="930" alt="Screenshot 2024-08-20 at 10 46 08 AM" src="https://github.com/user-attachments/assets/3a7ceb67-afaf-4d0b-af55-dde59bb63bb7">
<img width="2232" alt="Screenshot 2024-08-20 at 10 47 10 AM" src="https://github.com/user-attachments/assets/89292159-9356-44b9-8156-0da93da3bb2a">
Every time I rerun the app, there are new errors
<img width="935" alt="Screenshot 2024-08-20 at 10 49 40 AM" src="https://github.com/user-attachments/assets/501cf7b5-4fad-4294-9826-c5dafa63b392">
<img width="2225" alt="Screenshot 2024-08-20 at 10 55 49 AM" src="https://github.com/user-attachments/assets/9a9a1280-132f-4370-b0a6-63bb6e4eb262">

```
======== Exception caught by widgets library =======================================================
The following JSNoSuchMethodError was thrown building _FocusInheritedScope:
TypeError: Cannot read properties of undefined (reading 'Intl')

The relevant error-causing widget was: 
  MaterialApp MaterialApp:file:///.../.../.../lib/main.dart:79:28
When the exception was thrown, this was the stack: 
packages/flutter_gen/gen_l10n/app_localizations.dart 64:60               new
packages/flutter_gen/gen_l10n/app_localizations_en.dart 7:48             new
packages/flutter_gen/gen_l10n/app_localizations.dart 771:23              lookupAppLocalizations
packages/flutter_gen/gen_l10n/app_localizations.dart 756:48              load
packages/flutter/src/widgets/localizations.dart 57:48                    _loadAll
packages/flutter/src/widgets/localizations.dart 590:8                    load
packages/flutter/src/widgets/localizations.dart 554:5                    initState
packages/flutter/src/widgets/framework.dart 5748:55                      [_firstBuild]
packages/flutter/src/widgets/framework.dart 5593:5                       mount
packages/flutter/src/widgets/framework.dart 4468:15                      inflateWidget
packages/flutter/src/widgets/framework.dart 3963:18                      updateChild
packages/flutter/src/widgets/framework.dart 6900:14                      mount
packages/flutter/src/widgets/framework.dart 4468:15                      inflateWidget
packages/flutter/src/widgets/framework.dart 3957:20                      updateChild
...
```



Sometimes the app is just loading forever and nothing happens, then I see it in the browser 


```
Uncaught TypeError: Cannot read properties of undefined (reading 'Nested')
    at load__packages__provider__src__provider_dart (provider.dart.lib.js:230:63)
    at Object.execCb (require.js:1696:33)
    at Module.check (require.js:883:51)
    at Module.<anonymous> (require.js:1139:34)
    at require.js:134:23
    at require.js:1189:21
    at each (require.js:59:31)
    at Module.emit (require.js:1188:17)
    at Module.check (require.js:938:30)
    at Module.<anonymous> (require.js:1139:34)
    at require.js:134:23
    at require.js:1189:21
    at each (require.js:59:31)
    at Module.emit (require.js:1188:17)
    at Module.check (require.js:938:30)
    at Module.<anonymous> (require.js:1139:34)
    at require.js:134:23
    at require.js:1189:21
    at each (require.js:59:31)
    at Module.emit (require.js:1188:17)
    at Module.check (require.js:938:30)
    at Module.<anonymous> (require.js:1139:34)
    at require.js:134:23
    at require.js:1189:21
    at each (require.js:59:31)
    at Module.emit (require.js:1188:17)
    at Module.check (require.js:938:30)
    at Module.<anonymous> (require.js:1139:34)
    at require.js:134:23
    at require.js:1189:21
    at each (require.js:59:31)
    at Module.emit (require.js:1188:17)
    at Module.check (require.js:938:30)
    at Module.enable (require.js:1176:22)
    at Module.init (require.js:788:26)
    at callGetModule (require.js:1203:63)
    at Object.completeLoad (require.js:1590:21)
    at HTMLScriptElement.onScriptLoad (require.js:1717:29)

```
or this
    

 ```
   Uncaught TypeError: Cannot read properties of undefined (reading 'Symbol(_privateNames)')
    at dart.privateName (dart_sdk.js:56:24)
    at load__packages__graphql_flutter__src__widgets__graphql_provider_dart (graphql_provider.dart.lib.js:87:29)
    at Object.execCb (require.js:1696:33)
    at Module.check (require.js:883:51)
    at Module.<anonymous> (require.js:1139:34)
    at require.js:134:23
    at require.js:1189:21
    at each (require.js:59:31)
    at Module.emit (require.js:1188:17)
    at Module.check (require.js:938:30)
    at Module.enable (require.js:1176:22)
    at Module.init (require.js:788:26)
    at callGetModule (require.js:1203:63)
    at Object.completeLoad (require.js:1590:21)
    at HTMLScriptElement.onScriptLoad (require.js:1717:29)
```
And I also was getting errors provided above with Colors
    
Only after rerunning the app ~3-4 times, it's starting to work without the errors. Could you please help with the issue because it's really frustrating. Thank you.

**Comment by wartelski** [comment]
@huycozy could you please check ^^^

**Comment by hawkewebtpa** [comment]
Is this related or should I file a separate ticket for this variant?

Uncaught TypeError: Cannot read properties of undefined (reading 'load')
    at (index):207:30

We've run into similar issue. We recently tried upgrading our project from 3.19.5 to 3.24.0, but unfortunately when trying to implement update to deprecation in index.html for:
Warning: In index.html:189: "FlutterLoader.loadEntrypoint" is deprecated. Use "FlutterLoader.load" instead. See https://docs.flutter.dev/platform-integration/web/initialization for more details.
by changing:
...
_flutter.loader.loadEntrypoint({
...
to
...
_flutter.FlutterLoader.load({
...

Unfortunately causes issue with page just hanging on load, and only thing shown is in browser dev tool console:
Uncaught TypeError: Cannot read properties of undefined (reading 'load')
at (index):207:30

This goes away if I revert back to _flutter.loader.loadEntrypoint({.

Here is info with 3.24.0:
 flutter doctor -v
[✓] Flutter (Channel stable, 3.24.0, on macOS 14.5 23F79 darwin-arm64, locale en-US)
    • Flutter version 3.24.0 on channel stable at /Users/williamrobinson/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 80c2e84975 (3 weeks ago), 2024-07-30 23:06:49 +0700
    • Engine revision b8800d88be
    • Dart version 3.5.0
    • DevTools version 2.37.2

[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /Users/williamrobinson/Library/Android/sdk
    • Platform android-34, build-tools 34.0.0
    • ANDROID_HOME = /Users/williamrobinson/Library/Android/sdk
    • ANDROID_SDK_ROOT = /Users/williamrobinson/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.9+0-17.0.9b1087.7-11185874)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 15.4)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 15F31d
    • CocoaPods version 1.15.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2023.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.9+0-17.0.9b1087.7-11185874)

[✓] VS Code (version 1.90.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.94.0

[✓] Connected device (2 available)
    • Mac Designed for iPad (desktop) • mac-designed-for-ipad • darwin         • macOS 14.5 23F79 darwin-arm64
    • Chrome (web)                    • chrome                • web-javascript • Google Chrome 127.0.6533.120

[✓] Network resources
    • All expected network resources are available.

• No issues found!


And downgrading to 3.22.3 did not help (even after clean and rebuild), still same error:
run flutter doctor -v):
[✓] Flutter (Channel stable, 3.22.3, on macOS 14.5 23F79 darwin-arm64 (Rosetta), locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 15.4)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2023.2)
[✓] VS Code (version 1.90.2)
[✓] Connected device (2 available)
[✓] Network resources




Suggestions?

**Comment by huycozy** [comment]
@wartelski @hawkewebtpa Flutter stable hotfix 3.24.1 is just released. Please upgrade and retry.

**Comment by hawkewebtpa** [comment]
@huycozy  I tried the upgrade to 3.24.1, and still no joy. Created new ticket about the issue in case it is a different variant, with the details here:  https://github.com/flutter/flutter/issues/153941
